src/amo/components/App/index.js

```diff
@@ -20,6 +20,7 @@ import './styles.scss';
 import Routes from 'amo/components/Routes';
 import ScrollToTop from 'core/components/ScrollToTop';
 import NotAuthorizedPage from 'amo/pages/ErrorPages/NotAuthorizedPage';
+import NotAvailableInRegionPage from 'amo/pages/ErrorPages/NotAvailableInRegionPage';
```

Comment:
I'd stick to the http status code naming: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/451

Reply:
Good idea. I'll rename everything.